### PR TITLE
Changed Regexes

### PR DESCRIPTION
**Pull request by OlehHatsko**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Lunaflumen — APPROVED** [review]

File: EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs [hunk 1]
```diff
@@ -51,11 +51,11 @@ public class UserViewModel
         [MaxLength(50, ErrorMessage = "Адреса не може перевищувати 50 символів")]
         [Required(ErrorMessage = "Поле домашня адреса є обов'язковим")]
```
Comment (vovikboyko) [inline EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs:52]: Put field name in quotes

File: EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs [hunk 2]
```diff
@@ -51,11 +51,11 @@ public class UserViewModel
...
         public string Address { get; set; }
-        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`]{1,31}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`]{1,31})*$",
+        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,31}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,31})*$",
             ErrorMessage = "Псевдо має містити тільки літери")]
         [MaxLength(30, ErrorMessage = "Псевдо не може перевищувати 30 символів")]
         public string Pseudo { get; set; }
-        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`0-9.-]{1,51}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`0-9.-]{1,51})*$",
+        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`0-9.()-]{1,51}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`0-9.()-]{1,51})*$",
             ErrorMessage = "Поле Громадська, політична діяльність має містити тільки літери та цифри")]
         [MaxLength(50, ErrorMessage = "Поле Громадська, політична діяльність не може перевищувати 50 символів")]
```
Comment (vovikboyko) [inline EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs:60]: Put field name in quotes

File: EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs [hunk 3]
```diff
@@ -9,19 +9,19 @@ public class UserViewModel
...
+        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,26}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,26})*$",
             ErrorMessage = "Ім'я має містити тільки літери")]
         [Required(ErrorMessage = "Поле ім'я є обов'язковим")]
         [StringLength(25, MinimumLength = 2, ErrorMessage = "Ім'я повинне складати від 2 до 25 символів")]
         public string FirstName { get; set; }
         [Display(Name = "Прізвище")]
-        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`]{1,26}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`]{1,26})*$",
+        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,26}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,26})*$",
             ErrorMessage = "Прізвище має містити тільки літери")]
         [Required(ErrorMessage = "Поле прізвище є обов'язковим")]
```
Comment (vovikboyko) [inline EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs:20]: Put field name in quotes

File: EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs [hunk 4]
```diff
@@ -9,19 +9,19 @@ public class UserViewModel
         public string ID { get; set; }
         public string Email { get; set; }
         [Display(Name = "Ім'я")]
-        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`]{1,26}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'.`]{1,26})*$",
+        [RegularExpression(@"^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,26}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'().`]{1,26})*$",
             ErrorMessage = "Ім'я має містити тільки літери")]
         [Required(ErrorMessage = "Поле ім'я є обов'язковим")]
```
Comment (vovikboyko) [inline EPlast/EPlast.WebApi/Models/UserModels/UserViewModel.cs:14]: Put field name in quotes

**Review by vovikboyko — CHANGES_REQUESTED** [review]

**Review by vovikboyko — APPROVED** [review]

**Review by MahotskyiO — APPROVED** [review]

**Pull request merged**